rero_ils/modules/utils.py

```diff
@@ -419,3 +420,33 @@ def get_patron_from_arguments(**kwargs):
         or Patron.get_patron_by_barcode(kwargs.get('patron_barcode')) \
         or Patron.get_record_by_pid(kwargs.get('patron_pid')) \
         or Patron.get_record_by_pid(kwargs.get('loan').get('patron_pid'))
+
+
+def set_time_stamp(name, **kwargs):
+    """Set time stampe in current cache.
```

Comment:
typo : I thing that "timestamp" string is a not-splitted word (no need to have "_" between "time" and "stamp"
at L426 --> "stampe" should be "stamp"